plugins/woocommerce-admin/client/customize-store/intro/color-palettes.tsx

```diff
@@ -5,9 +5,17 @@ import { ColorPalette } from './types';
 
 export const ColorPalettes = ( {
 	colorPalettes,
+	totalPalettes,
 }: {
 	colorPalettes: ColorPalette[];
+	totalPalettes: number;
 } ) => {
+    let extra = null;
+
+    if ( totalPalettes > 4 ) {
```

Comment:
Ideally (to keep things simple), the total count can be retrieved with `colorPalettes.length` instead of using an additional field for it.

Reply:
Originally, I had included _all_ of the available color palettes in the array, so using `colorPalettes.length` wouldn't have worked. I realized there wasn't much need for passing color palette data we weren't using so dropped the array down to the 4 we're showing. Good point about keeping things simple though!